EIPS/eip-5757.md

```diff
@@ -0,0 +1,94 @@
+---
+eip: 5757
+title: Process for Approving External Resources
+description: Requirements and process for allowing new sources of external resources
+author: Sam Wilson (@SamWilsn)
+discussions-to: https://ethereum-magicians.org/t/eip-5757-process-for-approving-external-resources/11215
+status: Draft
+type: Meta
+created: 2022-09-30
+requires: 1
+---
+
+## Abstract
+
+Ethereum improvement proposals (EIPs) occasionally link to resources external to this repository. This document sets out the requirements for sources that may be linked to, and the process for approving a new source.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+### Definitions
+
+ - **Link**: Any method of referring to a resource, including: markdown links, anchor tags (`<a>`), images, citations of books/journals, and any other method of referencing content not in the current resource.
+ - **Resource**: A web page, document, file, or other media that contains content.
+ - **Source**: A publisher/chronicler of resources, like a standards body (eg. w3c) or a system of referring to documents (eg. Digital Object Identifier System).
+
+### Requirements for Sources
+
+Permissible sources **SHOULD NOT** obviously allow redistributing their resources as part of the EIP repository. If a source does obviously allow redistribution of resources, include those resources in an appropriate directory within this repository and link to those copies instead. The common exception to this rule are git repositories under the Ethereum organization.
+
+Permissible sources **MUST** provide a method of uniquely identifying a particular revision of a resource. Examples of such methods may include git commit hashes, version numbers, or publication dates.
+
+Permissible sources **MUST** have a proven history of availability. A source existing for at least ten years and reliably serving resources would be sufficient—but not necessary—to satisfy this requirement.
+
+Permissible sources **MUST NOT** charge a fee for accessing resources.
+
+### Source Removal
+
+Any approved source that ceases to satisfy the above requirements **MUST** be removed from [EIP-1](./eip-1.md), and its proposal (as described below) **MUST** be moved to a `status` of `Draft` or `Withdrawn`.
+
+Finalized EIPs (eg. those in the `Final` or `Withdrawn` statuses) **SHOULD NOT** be updated to remove links to these sources.
+
+Non-Finalized EIPs **MUST** remove links to these sources before changing statuses.
+
+### Source Approval
+
+#### Proposal
+
+Each potential permitted source **SHALL** be described in its own EIP.
```

Comment:
This is probably too hard to add a new source. And the granularity can be a debatable issue too.

Reply:
I've added a note to the rationale.